### DISCUSSION TITLE
Marketplace navigation bar pozition issue

### DISCUSSION THREAD

**Discussion by alikemalocalan** [community/community#19035] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi everyone,

i think marketplace natigation var is in wrong pozition on any browser, can you check it ?

https://github.com/marketplace?category=ides&type=actions

<img width="1171" alt="Screen Shot 2022-06-21 at 14 37 43" src="https://user-images.githubusercontent.com/3500876/174800821-0aba8791-7451-49a2-aa14-6b74d937a5f3.png">

**Comment by melvincwng** [comment, marked as answer]
Hi @alikemalocalan, thanks for informing the community about this issue.

I've tested it on my end using the link you've posted. On page 1, the navigation bar for Marketplace is situated on the left. However, on page 2, the navigation bar is situated in the center (more towards the right side of the screen). 

Due to this difference in the position of the navigation bar on both pages, it seems to me, that this could be a potential CSS issue/bug.

For bugs, you can report it to GitHub Support over [here](https://support.github.com/contact/bug-report). The support engineers will do an investigation, and would usually respond to you within a few working days.

Hope this helps.

**Reply by melvincwng** [reply]
@alikemalocalan, if you find this answer useful, please kindly 'Mark as answer' so that the others know that this issue has been addressed.

Appreciate it, thanks 🙏🏻
